sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -173,52 +181,472 @@ We guarantee that all client instance methods are thread-safe and independent of
+            {
+                Console.WriteLine("Item:");
+
+                if (itemField.ValueType == DocumentFieldType.Dictionary)
+                {
+                    IReadOnlyDictionary<string, DocumentField> itemFields = itemField.AsDictionary();
+
+                    if (itemFields.TryGetValue("Description", out DocumentField itemDescriptionField))
+                    {
+                        if (itemDescriptionField.ValueType == DocumentFieldType.String)
+                        {
+                            string itemDescription = itemDescriptionField.AsString();
+
+                            Console.WriteLine($"  Description: '{itemDescription}', with confidence {itemDescriptionField.Confidence}");
+                        }
+                    }
+
+                    if (itemFields.TryGetValue("Amount", out DocumentField itemAmountField))
+                    {
+                        if (itemAmountField.ValueType == DocumentFieldType.Double)
+                        {
+                            double itemAmount = itemAmountField.AsDouble();
+
+                            Console.WriteLine($"  Amount: '{itemAmount}', with confidence {itemAmountField.Confidence}");
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    if (document.Fields.TryGetValue("SubTotal", out DocumentField subTotalField))
+    {
+        if (subTotalField.ValueType == DocumentFieldType.Double)
+        {
+            double subTotal = subTotalField.AsDouble();
+            Console.WriteLine($"Sub Total: '{subTotal}', with confidence {subTotalField.Confidence}");
+        }
+    }
+
+    if (document.Fields.TryGetValue("TotalTax", out DocumentField totalTaxField))
+    {
+        if (totalTaxField.ValueType == DocumentFieldType.Double)
+        {
+            double totalTax = totalTaxField.AsDouble();
+            Console.WriteLine($"Total Tax: '{totalTax}', with confidence {totalTaxField.Confidence}");
+        }
+    }
+
+    if (document.Fields.TryGetValue("InvoiceTotal", out DocumentField invoiceTotalField))
+    {
+        if (invoiceTotalField.ValueType == DocumentFieldType.Double)
+        {
+            double invoiceTotal = invoiceTotalField.AsDouble();
+            Console.WriteLine($"Invoice Total: '{invoiceTotal}', with confidence {invoiceTotalField.Confidence}");
+        }
+    }
+}
+```
+
+You are not limited to receipts! There are a few prebuilt models to choose from, each of which has its own set of supported fields:
+- Analyze English business cards using the `prebuilt-businessCard` model. [Supported fields][businessCard_fields].
+- Analyze US driver licenses and international passports using the `prebuilt-idDocument` model. [Supported fields][idDocument_fields].
+- Analyze invoices using the `prebuilt-invoice` model. [Supported fields][invoice_fields].
+- Analyze English receipts using the `prebuilt-receipt` model. [Supported fields][receipt_fields].
```

Comment:
part of me prefers to remove English, Us, etc because if when the service starts supporting more languages, countries, the chances of us remembering to update this are very low

Reply:
I removed it only from invoice given your comment in the other PR, but ended up not paying attention to other prebuilts.

My major concern is having customers setting up a whole FR environment just to find out their documents are not supported. On the other hand, I'm not sure if we really have that many customers who heavily rely on our READMEs for documentation, so I could be overconcerning.

What do you think about removing all mentions to US/English, but always relying on the service's documentation right after that? We could update the sentence below to:
> For more samples and information about which types of documents are supported, see [here][analyze_prebuilt].

Reply:
>What do you think about removing all mentions to US/English, but always relying on the service's documentation right after that? We could update the sentence below to:
For more samples and information about which types of documents are supported, see [here][analyze_prebuilt].

like it!